collections/Lat_misc/MS_Lat_misc_e_125.xml

```diff
@@ -58,22 +326,22 @@
                      </origPlace>
                   </origin>
                   <provenance>
-                     <persName key="person_51680275">Prof. Hamilton Thompson</persName>
-                  </provenance>
+                     <persName key="person_51680275">Prof. A. Hamilton Thompson</persName>. A transcript by Hamilton Thompson is now MS. Lat. misc. e. 126: this includes a transcript of an eleventh folio, presently untraced, which contained the end of a further exemplum; an exemplum beginning 'Quidam frater inpugnabatur a fornicacione qui veniens in vicum quondam egipti', and ending 'et esto solicitus'; and another exemplum beginning 'Aliquando ostendit deus quare malis jus permittitur' and ending 'consolacionem scripturarum spem habeamus'.</provenance>
```

Comment:
Include these as `<msItem>`s above with an explanation that fol. 11 is now lost?